test/ch/aggregation_test.exs

```diff
@@ -69,8 +69,8 @@ defmodule Ch.AggregationTest do
     query = """
     select
     t.ticker as ticker,
-    toStartOfHour(t.time, 'UTC') as start_time,
-    toStartOfHour(t.time, 'UTC') + interval 1 hour as end_time,
+    toStartOfHour(t.time) as start_time,
+    toStartOfHour(t.time) + interval 1 hour as end_time,
```

Comment:
it would probably be `~N[]` now

Reply:
yeah but even then fails locally for me but lets try on CI

<img width="819" alt="image" src="https://user-images.githubusercontent.com/33396162/232325075-38ec208b-328d-4d2c-902e-d5c45af7dd9d.png">